app/mailers/review_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class ReviewMailer < ApplicationMailer
+  def new_review(review)
+    @review = review
+     mail(
+     	to: review.bourbon.user.email,
```

Comment:
Indent the first parameter one step more than the start of the previous line.<br>Tab detected.